shared/params/config.go

```diff
@@ -117,7 +117,7 @@ var defaultBeaconConfig = &BeaconChainConfig{
 	FarFutureEpoch:           1<<64 - 1,
 	BaseRewardsPerEpoch:      4,
 	DepositContractTreeDepth: 32,
-	MinGenesisDelay:          86400, // 1 day
+	GenesisDelay:             172800, // 1 day
```

Comment:
Please update comment.

```suggestion
	GenesisDelay:             172800, // 2 days
```